benchmark/compare.js

```diff
@@ -69,10 +75,24 @@ if (showProgress) {
 
 (function recursive(i) {
   const job = queue[i];
-
-  const child = fork(path.resolve(__dirname, job.filename), cli.optional.set, {
-    execPath: cli.optional[job.binary],
-  });
+  const resolvedPath = path.resolve(__dirname, job.filename);
+
+  const cpuCore = cli.getCpuCoreSetting();
+  let child;
+  if (cpuCore !== null) {
+    const spawnArgs = ['-c', cpuCore, cli.optional[job.binary], resolvedPath, ...cli.optional.set];
+    child = spawn('taskset', spawnArgs, {
+      env: process.env,
+      stdio: ['inherit', 'pipe', 'pipe'],
+    });
+
+    child.stdout.pipe(process.stdout);
+    child.stderr.pipe(process.stderr);
```

Comment:
Looking back at it, this should not be done as you mess with the stdout of this process which should be CSV

Reply:
ohh I wanted to console logs back to parent that's why I added these pipes

Reply:
Can you open a fresh issue/pr to fix this?